app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/ui/InsuranceContractExt.kt

```diff
@@ -0,0 +1,53 @@
+package com.hedvig.android.feature.insurances.ui
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.ColorPainter
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.hedvig.android.core.ui.insurance.toDrawableRes
+import com.hedvig.android.feature.insurances.data.InsuranceContract
+import hedvig.resources.R
+import kotlinx.collections.immutable.ImmutableList
+import kotlinx.collections.immutable.toPersistentList
+import kotlinx.datetime.Clock
+import kotlinx.datetime.TimeZone
+import kotlinx.datetime.todayIn
+
+@Composable
+fun InsuranceContract.createChips(): ImmutableList<String> {
```

Comment:
Could it also be `internal` if we only use it in this module?

But yeah I *feel* like this is a bit nicer, since we do not need to add this code so close to the data layer. And no reference to context so close to the data layer either. Let's see if it proves to be harder to maintain or problematic if we need to use this in multiple places and so on.